definitions/procedures/pulp/remove.rb

```diff
@@ -24,7 +24,15 @@ def pulp_data_dirs
         '/var/lib/pulp/uploads',
         '/var/lib/mongodb/',
         '/var/cache/pulp'
-      ]
+      ].select { |dir| File.directory?(dir) }
```

Comment:
What is this `directory?` for? I know it was there before, just not understanding the point.